core/arch/aarch64/proc.c

```diff
@@ -45,8 +45,14 @@ proc_init_arch(void)
     num_simd_registers = MCXT_NUM_SIMD_SLOTS;
     num_opmask_registers = MCXT_NUM_OPMASK_SLOTS;
 
-    set_cache_line_size_using_ctr_el0(/* dcache_line_size= */ &cache_line_size,
-                                      /* icache_line_size= */ NULL);
+    size_t dcache_line_size;
+    // When DR_HOST_NOT_TARGET, get_cache_line_size returns false and does
+    // not set any value in given args. In that case, we do not want to
```

Comment:
But if it doesn't set the arg, we can just pass in cache_line_size.  No need for a separate local.

Reply:
Done.

For some reason, I thought it safer to do it that way. But I guess it's okay.